src/lib.js

```diff
@@ -32,9 +32,9 @@ const getExecutedScript = () => {
  * @param  {Element} tag The host element
  * @return {Object}  props object to be passed to the component
  */
-const collectPropsFromElement = element => {
+const collectPropsFromElement = (element, props = Object.create({})) => {
```

Comment:
this one can also be `props = {}`. why Object.create ?

Reply:
yeah it looks like its a little bit useless to use the Object.create I agree, hopefully @heldr can give his opinion about it.